database/things/elixir.pldb

```diff
@@ -4,7 +4,6 @@ type pl
 creators José Valim
 website https://elixir-lang.org
 fileExtensions ex exs
-corporateDevelopers plataformatec
```

Comment:
Elixir was backed by Plataformatec for several years but the company no longer exists. :)